src/test/java/io/cucumber/skeleton/StepDefinitions.java

```diff
@@ -1,11 +1,23 @@
 package io.cucumber.skeleton;
 
 import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
 
 public class StepDefinitions {
     @Given("I have {int} cukes in my belly")
     public void I_have_cukes_in_my_belly(int cukes) {
         Belly belly = new Belly();
         belly.eat(cukes);
     }
+
+    @When("I wait {int} hour")
+    public void iWaitHour(int hCount) {
+        System.out.println("I wait for " + hCount + " hour");
```

Comment:
I'm not sure about this. There either should be some sort of example implementation here or we should allow the tests to fail.

Reply:
I didn't intend to add these step definitions at first, but lacking of these causes Travis CI to [failed](https://travis-ci.org/github/cucumber/cucumber-java-skeleton/builds/697920497?utm_source=github_status&utm_medium=notification). Is it acceptable?

Reply:
The build should not fail, but we can tell travis to run maven and allow the tests to fail by running `mvn test -Dmaven.test.failure.ignore=true`.

Reply:
hmmm, should I revert this definitions, since dealing with Travis is out of this PR scope?

Reply:
That'd be preferable. I can look at travis.

Reply:
I reverted this stuffs, please take a look and resolve conversation if you have no further comments @mpkorstanje :D